README.md

```diff
@@ -5,6 +5,18 @@ A Firefox userChrome.css theme that aims to recreate the look and feel of the Ch
 <img src="screenshots/thumbnail.png" alt="thumbnail screenshot" title="Screenshot taken with macOS Monterey / Firefox Nightly 96.0a1 (2021-11-30)" width="800">
 
 ## How to install
+### Via Linux and MacOS shell script
+Paste this into your terminal emulator:
+```sh
+# Linux may not have $TMPDIR set by default
+if [ -z $TMPDIR ]; then TMPDIR=/tmp; fi;
+
+curl https://raw.githubusercontent.com/bmFtZQ/edge-frfox/blob/main/install.sh > $TMPDIR/installer.sh && chmod +x $TMPDIR/installer.sh && sh $TMPDIR/installer.sh
```

Comment:
FYI this won't work until it exists in the main branch, for now you can test it with:

```sh
# Linux may not have $TMPDIR set by default
if [ -z $TMPDIR ]; then TMPDIR=/tmp; fi;

curl https://raw.githubusercontent.com/bmFtZQ/edge-frfox/bcf1ea15fd7cecddeda0e3cadde5c5626b9d8da0/install.sh > $TMPDIR/installer.sh && chmod +x $TMPDIR/installer.sh && sh $TMPDIR/installer.sh
```